pom.xml

```diff
@@ -120,8 +120,8 @@
         <jena.version>3.9.0</jena.version>
         <jena.fuseki.version>3.8.0</jena.fuseki.version>
         
-        <fcrepo-camel.version>4.5.0</fcrepo-camel.version>
-        <camel.version>2.22.1</camel.version>
+        <fcrepo-camel.version>5.0.0</fcrepo-camel.version>
+        <camel.version>2.20.4</camel.version>
```

Comment:
Why was camel downgraded?

Reply:
Good point. When I was working on this I tried updating camel to 2.24.1 since it was the most reason version, but it didn't work because camel 2.23 stopped supporting spring 4. Apparently i then decided to set it to the exact version of camel being using by fcrepo-camel, but there's no need to do that. I've updated it to 2.22.5